<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLCDNumber" name="display_lbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="division_btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        font: 24pt &quot;MS Shell Dlg 2&quot;;
		font: 24pt &quot;MS Shell Dlg 2&quot;;
		background-color: rgb(88, 191, 255);
  }
 QPushButton:hover {
       background-color: yellow;
		color: black;
 }</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="multiplication_btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        font: 24pt &quot;MS Shell Dlg 2&quot;;
		font: 24pt &quot;MS Shell Dlg 2&quot;;
		background-color: rgb(88, 191, 255);
  }
 QPushButton:hover {
       background-color: yellow;
		color: black;
 }</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="subtract_btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        font: 24pt &quot;MS Shell Dlg 2&quot;;
		font: 24pt &quot;MS Shell Dlg 2&quot;;
		background-color: rgb(88, 191, 255);
  }
 QPushButton:hover {
       background-color: yellow;
		color: black;
 }</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>41</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        font: 24pt &quot;MS Shell Dlg 2&quot;;
		font: 24pt &quot;MS Shell Dlg 2&quot;;
		background-color: rgb(88, 191, 255);
  }
 QPushButton:hover {
       background-color: yellow;
		color: black;
 }</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="equals_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        font: 24pt &quot;MS Shell Dlg 2&quot;;
		font: 24pt &quot;MS Shell Dlg 2&quot;;
		background-color: rgb(88, 191, 255);
  }
 QPushButton:hover {
       background-color: yellow;
		color: black;
 }</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 26, 6);
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 0), stop:0.52 rgba(0, 0, 0, 0), stop:0.565 rgba(82, 121, 76, 33), stop:0.65 rgba(159, 235, 148, 64), stop:0.721925 rgba(255, 238, 150, 129), stop:0.77 rgba(255, 128, 128, 204), stop:0.89 rgba(191, 128, 255, 64), stop:1 rgba(0, 0, 0, 0));</string>
   </property>
   <property name="text">
    <string>#preeminence</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        font: 24pt &quot;MS Shell Dlg 2&quot;;
		color: rgb(255, 255, 255);
		background-color: rgb(0, 113, 0);
  }
 QPushButton:hover {
       background-color: yellow;
		color: black;
 }</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        font: 24pt &quot;MS Shell Dlg 2&quot;;
		color: rgb(255, 255, 255);
		background-color: rgb(0, 113, 0);
  }
 QPushButton:hover {
       background-color: yellow;
		color: black;
 }</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        font: 24pt &quot;MS Shell Dlg 2&quot;;
		color: rgb(255, 255, 255);
		background-color: rgb(0, 113, 0);
  }
 QPushButton:hover {
       background-color: yellow;
		color: black;
 }</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        font: 24pt &quot;MS Shell Dlg 2&quot;;
		color: rgb(255, 255, 255);
		background-color: rgb(0, 113, 0);
  }
 QPushButton:hover {
       background-color: yellow;
		color: black;
 }</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_0">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        font: 24pt &quot;MS Shell Dlg 2&quot;;
		color: rgb(255, 255, 255);
		background-color: rgb(0, 113, 0);
  }
 QPushButton:hover {
       background-color: yellow;
		color: black;
 }</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        font: 24pt &quot;MS Shell Dlg 2&quot;;
		color: rgb(255, 255, 255);
		background-color: rgb(0, 113, 0);
  }
 QPushButton:hover {
       background-color: yellow;
		color: black;
 }</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        font: 24pt &quot;MS Shell Dlg 2&quot;;
		color: rgb(255, 255, 255);
		background-color: rgb(0, 113, 0);
  }
 QPushButton:hover {
       background-color: yellow;
		color: black;
 }</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        font: 24pt &quot;MS Shell Dlg 2&quot;;
		color: rgb(255, 255, 255);
		background-color: rgb(0, 113, 0);
  }
 QPushButton:hover {
       background-color: yellow;
		color: black;
 }</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        font: 24pt &quot;MS Shell Dlg 2&quot;;
		color: rgb(255, 255, 255);
		background-color: rgb(0, 113, 0);
  }
 QPushButton:hover {
       background-color: yellow;
		color: black;
 }</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        font: 24pt &quot;MS Shell Dlg 2&quot;;
		color: rgb(255, 255, 255);
		background-color: rgb(0, 113, 0);
  }
 QPushButton:hover {
       background-color: yellow;
		color: black;
 }</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        font: 24pt &quot;MS Shell Dlg 2&quot;;
		font: 24pt &quot;MS Shell Dlg 2&quot;;
		background-color: grey;
  }
 QPushButton:hover {
       background-color: yellow;
		color: black;
 }</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        font: 24pt &quot;MS Shell Dlg 2&quot;;
		font: 24pt &quot;MS Shell Dlg 2&quot;;
		background-color: grey;
  }
 QPushButton:hover {
       background-color: yellow;
		color: black;
 }</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="float_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        font: 24pt &quot;MS Shell Dlg 2&quot;;
		font: 24pt &quot;MS Shell Dlg 2&quot;;
		background-color: grey;
  }
 QPushButton:hover {
       background-color: yellow;
		color: black;
 }</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="log_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>41</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		font: 12pt &quot;MS Shell Dlg 2&quot;;
		background-color: grey;
  }
 QPushButton:hover {
       background-color: yellow;
		color: black;
 }</string>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
